src/components/license-activation/LicenseActivationPage.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { EnterprisePage } from '../enterprise-page';
+import { EnterpriseBanner } from '../enterprise-banner';
+import { Layout } from '../layout';
+import LicenseActivation from './LicenseActivation';
+
+const ActivationPage = () => (
+  <EnterprisePage>
+    <Layout>
+      <EnterpriseBanner />
+      <LicenseActivation />
```

Comment:
If this page will allow for unauthenticated access, will we be able to render `EnterprisePage`, `Layout`, and `EnterpriseBanner` since there are some API requests within `EnterprisePage` and the other components depend on those?

Reply:
This page doesn't really allow for unauthenticated access, as the user will be immediately redirected to the proxy login view if they're not already authenticated. That will happen after `<LoginRedirect />` is added to the page though

Reply:
Ah, right. Great. That resolves this question :)